src/components/App.js

```diff
@@ -57,31 +75,57 @@ class App extends Component {
     return this.state.notes[index]
   }
 
-  save () {
+  saveNotes () {
     localStorage.setItem('Notestack', JSON.stringify(this.state.notes))
   }
 
-  load () {
+  saveTags () {
+    localStorage.setItem('Notestack-Tags', JSON.stringify(this.state.tags))
+  }
+
+  saveKeywords () {
+    localStorage.setItem('Notestack-Keywords', JSON.stringify(this.state.keywords))
+  }
+
+  loadNotes () {
     try {
       return JSON.parse(localStorage.getItem('Notestack')) || dummyNotes
     } catch (err) {
       return dummyNotes
     }
   }
 
+  loadTags () {
+    try {
+      return JSON.parse(localStorage.getItem('Notestack-Tags')) || dummyTags
+    } catch (err) {
+      return dummyTags
+    }
+  }
+
+  loadKeywords () {
+    try {
+      return JSON.parse(localStorage.getItem('Notestack-Keywords')) || []
+    } catch (err) {
+      return []
+    }
+  }
+
   render () {
-    this.save()
+    this.saveNotes()
+    this.saveTags()
```

Comment:
verschiebe bitte diese beiden Funktionen in `componentDidUpdate`

Reply:
done